browser/src/layer/tile/CanvasTileLayer.js

```diff
@@ -2014,43 +2014,23 @@ L.CanvasTileLayer = L.Layer.extend({
 	},
 
 	_onCalcFunctionUsageMsg: function (textMsg) {
-		var pos = this._lastVisibleCursorRef._northEast;
-		this._map.uiManager.showFormulaTooltip(textMsg, pos);
+		var formulaAutocompletePopup = L.DomUtil.get('formulaautocompletePopup');
+		if (formulaAutocompletePopup)
+			this._map.fire('closeformulapopup');
```

Comment:
do we need to check if it exists here? can we just fire the event, and check if it exists inside event handler? (so we will not repeat that code again)

Reply:
Done with https://github.com/CollaboraOnline/online/pull/9201 New PR because it decided not merge 23.05 branch So I'll abort this PR after the conversation.